modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -128,7 +128,7 @@ EXTERNAL-IP   OS-IMAGE                                       	KERNEL-VERSION
 CONTAINER-RUNTIME
 rt-worker-0.example.com	          Ready	 worker,worker-rt   5d17h   v1.22.1
 128.66.135.107   <none>    	        Red Hat Enterprise Linux CoreOS 46.82.202008252340-0 (Ootpa)
-4.18.0-211.rt5.23.el8.x86_64   cri-o://1.22.1-90.rhaos4.9.git4a0ac05.el8-rc.1
+4.18.0-211.rt5.23.el8.x86_64   cri-o://1.22.1-90.rhaos4.10.git4a0ac05.el8-rc.1
```

Comment:
```suggestion
4.18.0-305.30.1.rt7.102.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
```

Reply:
Also update [this line](https://github.com/openshift/openshift-docs/pull/39295/files/28f830dbc189ae28405bff6432a55f7a63a563f7#diff-cb6107e6ffe4c7fb5ef9ebd3ecccc67dc682f0666a4bc337a85b9f42353b2047R122) as follows: 

```
Note the worker with the role `worker-rt` that contains the string `4.18.0-305.30.1.rt7.102.el8_4.x86_64`:
```

Reply:
@MarSik how does this look?